### PR TITLE
New signature to detect cscript/wscript dropper using the sysmon file event

### PR DESCRIPTION
**Pull request by redsand**
Relates to existing signature but does not focus on command execution

### CODE REVIEW

File: rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+status: experimental
+description: Detects a file ending in jse, vbe, js, vba, vbs written by cscript.exe or wscript.exe
+related: 
+    - id: cea72823-df4d-4567-950c-0b579eaf0846
+      type: derived
+references:
+    - WScript or CScript Dropper (cea72823-df4d-4567-950c-0b579eaf0846)
+author: Tim Shelton
+date: 2022/01/10
+modified: 2022/01/10
```
Comment (frack113) [inline rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml:12]: HI no need to add `modified` it is the first release (can not be modified by someone)

File: rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        TargetFilename|startswith: 
+            - 'C:\Users\'
+            - 'C:\ProgramData'
+        TargetFilename|endswith: 
+            - '.jse'
+            - '.vbe'
+            - '.js'
+            - '.vba'
+            - '.vbs'
+    condition: 1 of selection*
```
Comment (frack113) [inline rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml:30]: As there is only 1 selection ( Image AND TargetFilename AND TargetFilename) simply `condition:  selection1`

**Review by frack113 — CHANGES_REQUESTED** [review]

File: rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml (hunk 1 above)
Comment (redsand) [reply rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml:12]: Reckon it's confusing for new people when updating it, to also remember to add a modified field, vis it already being present.  I think this is 6 one way and half a dozen another

File: rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml (hunk 2 above)
Comment (redsand) [reply rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml:30]: Agreed, only if targetfilename is the only field and will be the only field for the foreseeable future.

Comment (redsand) [reply rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml]: this has been completed

File: rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml (hunk 1 above)
Comment (redsand) [reply rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml]: modified has been left in, since my changes span 2 days, hope thats ok.

**Review by frack113 — APPROVED** [review]

**Comment by Neo23x0** [comment]
This pull request also contains changes to the hawk backend. Is this intended? 
If yes, could we do this in two different pull requests. (rules changes <> backend changes)

**Comment by redsand** [comment]
> This pull request also contains changes to the hawk backend. Is this intended? If yes, could we do this in two different pull requests. (rules changes <> backend changes)

Not intended! good catch. let me clean up, sec. I have a habit of using git commit -a

**Comment by redsand** [comment]
This has been resolved, thanks!

**Comment by Neo23x0** [comment]
No problem. Thanks for the rule.

**Pull request merged**